scripts/.build_vars

```diff
@@ -7,6 +7,7 @@ export LIB_DIR=${EOSIO_INSTALL_DIR}/lib
 export DATA_DIR=${EOSIO_INSTALL_DIR}/data
 
 # CMAKE
+export CMAKE_REQUIRED_VERSION=$(cat $REPO_ROOT/CMakeLists.txt | grep -E "cmake_minimum_required\([[:blank:]]*VERSION" | tail -1 | sed 's/.*VERSION //g' | cut -d\) -f1 | sed 's/ //g')
```

Comment:
This could safely go into `.environment` instead so it is only called once in the script.

Also for further resilience it could be:
```
export CMAKE_REQUIRED_VERSION=$(cat $REPO_ROOT/CMakeLists.txt | grep -E "^[[:blank:]]*cmake_minimum_required[[:blank:]]*\([[:blank:]]*VERSION" | tail -1 | sed 's/.*VERSION //g' | sed 's/ //g' | sed 's/"//g' | cut -d\) -f1)
```